src/components/JobHeader.vue

```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import { ref, onMounted } from 'vue'
 import { useRoute, RouterLink } from 'vue-router'
-import { getHeaderById } from '@/api/schedules/jobposting/jobposting'
+import { getHeaderById } from '@/api/job-posting/index'
```

Comment:
```suggestion
import { getHeaderById } from '@/api/job-posting'
```

디렉토리아래에 이름이 index.ts인 파일은 import 시에 index를 생략할 수 있어요!